File: QolmsYappli/QolmsYappli/Models/InnerModel/ConnectionSettingHospitalItem.vb
''' <summary>
''' 病院の連携設定情報を表します。
''' このクラスは継承できません。
''' </summary>
''' <remarks></remarks>
<Serializable()>
Public NotInheritable Class ConnectionSettingHospitalItem

#Region "Public Property"

    ''' <summary>
    ''' 連携番号を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property LinkageSystemNo As Integer = Integer.MinValue

    ''' <summary>
    ''' 連携番号を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property LinkageSysyemName As String = String.Empty

    ''' <summary>
    ''' 連携 ステータスを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property Status As Byte = Byte.MinValue




#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="CouponItem" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
    End Sub

#End Region

End Class

File: QolmsJotoWebView/Models/InnerModels/ConnectionSettingHospitalItem.cs
using System;

namespace MGF.QOLMS.QolmsJotoWebView
{
    /// <summary>
    /// 病院の連携設定情報を表します。
    /// </summary>
    [Serializable]
    public sealed class ConnectionSettingHospitalItem
    {
        public int LinkageSystemNo { get; set; } = int.MinValue;

        public string LinkageSystemName { get; set; } = string.Empty;

        public byte Status { get; set; } = byte.MinValue;
    }
}